internal/git/git.go

```diff
@@ -69,10 +69,10 @@ func (g *Git) Worktree(
 	version string,
 	dstDir string,
 ) error {
-	dst, err := g.appFs.Abs(dstDir)
-	if err != nil {
-		return err
-	}
+	// Ignoring errors as the only time `Abs` will return an error is when
+	// `os.Getwd()` returns an error.  This situation wouldn't happen, and
+	// proves difficult to test.
+	dst, _ := g.appFs.Abs(dstDir)
```

Comment:
I mean, it's a weird, wild world out there, so anything is possible.  But to your point, the MemFS we use for unit tests literally cannot return an error from `Abs`.

I've been noodling over how plausible it would be to implement a "FailFS" fixture to put over the top of the MemFS, kind of like what their ReadOnlyFS does, so we could mock up all sorts of weird scenarios to get all the code paths under test.

If such things come to pass, this change could always be backed out.

Reply:
Good points, crudely tested this to retain error checking, until we get a better soltuion.